dotnet-desktop-guide/net/winforms/controls-design/overview.md

```diff
@@ -56,61 +72,44 @@ Deciding which kind of control to create can be confusing. This article highligh
 
-Creating a control by inheriting from the <xref:System.Windows.Forms.Control> class requires much more thought and effort than inheriting from <xref:System.Windows.Forms.UserControl> or an existing Windows Forms control. Because a great deal of implementation is left for you, your control can have greater flexibility than a composite or extended control, and you can tailor your control to suit your exact needs.
+Creating a control by inheriting from the <xref:System.Windows.Forms.Control> class requires more thought and effort than inheriting from <xref:System.Windows.Forms.UserControl> or an existing Windows Forms control. Because a great deal of implementation is left for you, your control can have greater flexibility than a composite or extended control, and you can tailor your control to suit your exact needs.
 
-To implement a custom control, you must write code for the <xref:System.Windows.Forms.Control.OnPaint%2A> event of the control, as well as any feature-specific code you need. You can also override the <xref:System.Windows.Forms.Control.WndProc%2A> method and handle windows messages directly. This is the most powerful way to create a control, but to use this technique effectively, you need to be familiar with the Microsoft Win32® API.
+To implement a custom control, you must write code for the <xref:System.Windows.Forms.Control.OnPaint%2A> event of the control, which controls how the control is visually drawn. You must also write any feature-specific behaviors for your control. You can also override the <xref:System.Windows.Forms.Control.WndProc%2A> method and handle windows messages directly. This is the most powerful way to create a control, but to use this technique effectively, you need to be familiar with the Microsoft Win32® API.
 
 An example of a custom control is a clock control that duplicates the appearance and behavior of an analog clock. Custom painting is invoked to cause the hands of the clock to move in response to <xref:System.Windows.Forms.Timer.Tick> events from an internal <xref:System.Windows.Forms.Timer> component.<!-- TODO For more information, see [How to: Develop a Simple Windows Forms Control](how-to-develop-a-simple-windows-forms-control.md).-->
 
-## ActiveX Controls
-
-Although the Windows Forms infrastructure has been optimized to host Windows Forms controls, you can still use ActiveX controls. There's support for this task in Visual Studio.<!-- TODO For more information, see [How to: Add ActiveX Controls to Windows Forms](how-to-add-activex-controls-to-windows-forms.md).-->
+## ActiveX controls
 
-## Windowless Controls
+Although the Windows Forms infrastructure has been optimized to host Windows Forms controls, you can still use ActiveX controls. <!--TODO There's support for this task in Visual Studio. For more information, see [How to: Add ActiveX Controls to Windows Forms](how-to-add-activex-controls-to-windows-forms.md).-->
```

Comment:
Well...we cannot say that right now, since we don't know. We might need to change the runtime, but then we also need to decide, if we wanted to service that back to .NET 6 and 7 (if we'd make the bar for that). We hope that updating the Designer will do it (so, it would be surfacing in 17.7), but we're not sure yet. @merriemcgaw FYI.

Reply:
@merriemcgaw, what's your take? Technically, they aren't yet for .NET, and only in beta for Framework OOP.
For transparency, we should mention the current status quo, would be my suggestion.

Reply:
We've done a lot of clean up of the ActiveX code in the .NET 8 timeframe. Before that I'd call it "as is" 😄 . For 17.7 we expect the experience of adding these to the form and interacting with it. This would be both with the .NET out of process designer and the .NET Framework support in the out of process designer. Nothing was blocking it previously really in the run time, it's just that until 17.7 we didn't have the axHost control designer on the .NET toolbox. Adding @Shyam-Gupta to confirm the experience today and 17.7 GA.

Side note @adegeo - we've got to get some documentation created soon around 32bit references in VS2022 WinForms applications. We're working on what we want the experience to be then we'll have an urgent doc request once we're there. I'll keep you posted on that front.